src/component/Autocomplete/Autocomplete.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState, useMemo, useRef, useEffect } from 'react';
+import debounce from 'lodash.debounce';
```

Comment:
Do not forgot commit dependecies (package.json + package-lock.json files)
![2024-03-13_16h30_09](https://github.com/mate-academy/react_autocomplete/assets/61904995/66a0d8ad-edcf-4639-8c37-3a5b850ece0f)